src/app/services/locales/en-us/index.yaml

```diff
@@ -23,14 +23,27 @@ services:
       config: 'YAML'
     items:
       title: Services
-      intro: !!text/markdown |
-        A Service represents an application or microservice that is registered with the mesh and can be managed, monitored and secured through the mesh's policies.
       navigation:
-        mesh-service-list-view: MeshService
-        mesh-multi-zone-service-list-view: MeshMultiZoneService
-        mesh-external-service-list-view: MeshExternalService
-        service-list-view: Internal
-        external-service-list-view: External
+        mesh-service-list-view:
+          label: MeshService
+          description: !!text/markdown |
+            A <a href="https://kuma.io/docs/latest/networking/meshservice/#meshservice" target="_blank">MeshService</a> represents a destination for traffic from elsewhere in the mesh and can define several networking details. The behaviour of this resource depends on the zone type. 
+        mesh-multi-zone-service-list-view:
+          label: MeshMultiZoneService
+          description: !!text/markdown |
+            A <a href="https://kuma.io/docs/latest/networking/meshmultizoneservice/#meshmultizoneservice" target="_blank">MeshMultiZoneService</a> represents a group of `MeshService` resources in a loadbalanced multizone deployment. `MeshService` resources that are deployed across several clusters can be grouped by a `MeshMultiZoneService`.
```

Comment:
nit: I'm no subject matter expert here, but I wondered why we used the term `clusters` here instead of `zones`, should we use `zones` or maybe something like `zones/clusters`? Maybe the answer to this question, is just a "nope, clusters is exactly the term we want to use"

Reply:
Hmmm 🤔 I'm also not an expert, but I took this wording from the [docs](https://kuma.io/docs/latest/networking/meshmultizoneservice/#meshmultizoneservice)
> The most common use case for using MeshMultiZoneService is to group MeshServices that spawn across multiple clusters with failover to another cluster in case of unavailability in the local cluster.

Reply:
Cool, lets resolve this one then